src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -8,6 +8,7 @@
     "apache-airflow": ("airflow",),
     "attrs": ("attr",),
     "beautifulsoup4": ("bs4",),
+    "BitVector": ("BitVector",),
```

Comment:
The left-side needs to be normalized per https://www.python.org/dev/peps/pep-0503/#normalized-names.

```suggestion
    "bitvector": ("BitVector",),
```

Reply:
Good thing there's a test for that :)

Reply:
I actually originally started with the change you suggested, but it didn't work when I added it to my top level BUILD file (I get `ModuleNotFoundError`).

This works:
```
python_requirements(
  module_mapping={
    "BitVector": ["BitVector"],
  },
)
```

The change I proposed worked in testing. So I'm not sure - should the format be different in the default_module_mapping, or is it also not going to work?

Reply:
I'm guessing that you also have "BitVector", in your requirements.txt file?
If you also change that to be lowercase, I guess it will work with Eric's suggested change..?

Reply:
Ah.. got it, thanks!

Reply:
Ah ha! You found a new bug. This should be resilient to the case you're describing. It is with DEFAULT_MODULE_MAPPING, but not `python_requirements`. Thanks for the report! Fixing now.